src/util/defs.h

```diff
@@ -1,7 +1,11 @@
 #pragma once
 
+#include "preferences/configobject.h"
+
 // Maximum buffer length to each EngineObject::process call.
 //TODO: Replace this with mixxx::AudioParameters::bufferSize()
 constexpr unsigned int MAX_BUFFER_LEN = 160000;
 
 constexpr int kMaxNumberOfDecks = 4;
+
+const ConfigKey kMaximizeLibraryConfigKey = ConfigKey("[Library]", "maximize_library");
```

Comment:
Let's reduce redundancy if we rename it anyway. 

Also, the active verb in `maximize_library` makes it sound like a button (e.g. `[ChannelN],hotcue_X_activate`), which means that you have to press [1.0] and release [0.0] to enable, the press and release again to disable.

But in this case it's actually a flag (1.0 = enabled, 0.0 = disabled). So I think we should use a passive verb here do indicate that (like we do in `[ChannelN],loop_enabled`). 

```suggestion
const ConfigKey kMaximizeLibraryConfigKey = ConfigKey("[Library]", "maximized");
```

Reply:
I see what you mean, but I think for consistency we should either name it  like `show_previewdeck` and other visibility controls -- or rename all.
Btw despite the redundancy the benefit of `maximize_library` is that the name of the control matches the tooltip which is convenient for skin development.

And actually the reason for using the `[Library]` is solely that it's listed in the respective chapter in the manual, see the bug report. `[Skin],maxmize_library` would be more appropriate IMO.
On that topic: I think we should rename the chapters to `Deck controls`, `Skin controls` etc., much like the submenus in the control picker menu. Also, the `Group` elements in the chapters do decrease the readability IMHO, so I'd appreciate if we'd rename them.

Reply:
> rename the chapters to `Deck controls`, `Skin controls` etc., much like the submenus in the control picker menu.

![image](https://user-images.githubusercontent.com/5934199/136012366-c46799dd-8f62-49a5-b25e-3c6c82179d15.png)
vs.
![image](https://user-images.githubusercontent.com/5934199/136014902-3a29ecba-2a68-47b2-988d-29b42881989b.png)